hooks/ciliumnetworkpolicy_util.go

```diff
@@ -57,7 +57,9 @@ func (v *ciliumNetworkPolicyValidator) toIPNetSlice(raw []string) ([]*net.IPNet,
 func (v *ciliumNetworkPolicyValidator) gatherIPFilters(nparl *tenetv1beta2.NetworkPolicyAdmissionRuleList, ls map[string]string) ([]*net.IPNet, []*net.IPNet, error) {
 	var egressFilters, ingressFilters []*net.IPNet
 	for _, npar := range nparl.Items {
-		if !v.shouldValidate(&npar, ls) {
+		if ok, err := v.shouldExclude(&npar, ls); err != nil {
```

Comment:
`ok` may be confusing.
How about using `matched`?

Reply:
Looks good. I'll fix it.